### PR TITLE
Added all inifile templates and fixed bugs

### PR DESCRIPTION
**Pull request by sainho93**

### CODE REVIEW

File: Resources/resources.qrc [hunk 1]
```diff
@@ -74,6 +74,43 @@
...
+        <file>inifile_templates/test_8_gcfm.xml</file>
+        <file>inifile_templates/test_6_tordeux2015.xml</file>
+        <file>inifile_templates/test_5_tordeux2015.xml</file>
+        <file>inifile_templates/test_4_tordeux2015.xml</file>
+        <file>inifile_templates/test_4_gompertz.xml</file>
+        <file>inifile_templates/test_3_tordeux2015.xml</file>
+        <file>inifile_templates/test_2_tordeux2015.xml</file>
+        <file>inifile_templates/test_4_gcfm.xml</file>
+        <file>inifile_templates/test_2_gompertz.xml</file>
+        <file>inifile_templates/test_2_gcfm.xml</file>
```
Comment (zeroset) [inline Resources/resources.qrc]: Could you order them by test number? So that we have a better overview?

File: src/datamanager.cpp [hunk 2]
```diff
@@ -1027,8 +1030,37 @@ void jpsDatamanager::writeCrossings(QXmlStreamWriter *stream, JPSZone *room, QLi
...
+            {
+                case QMessageBox::Ok:
+                    // Ok was clicked
+                    break;
+                case QMessageBox::Cancel:
+                    break;
+                default:
+                    // should never be reached
+                    break;
+            }
```
Comment (zeroset) [inline src/datamanager.cpp]: What is this switch for?

File: src/widgets/templatewidget.cpp [hunk 3]
```diff
@@ -56,7 +56,7 @@ void TemplateWidget::pushButton_chooseClicked()
     qDebug("Enter TemplateWidget::pushButton_chooseClicked");
     int current_int = ui->treeWidget->currentColumn();
     QString current_name = ui->treeWidget->currentItem()->text(current_int);
-    sendTemplateName(current_name);
+    emit sendTemplateName(current_name);
```
Comment (zeroset) [inline src/widgets/templatewidget.cpp]: What is this emit?

File: Resources/resources.qrc (hunk 1 above)
Comment (sainho93) [reply Resources/resources.qrc]: I'va added then by Qt Creator, in the widget they shows by test number, the order in .qrc won't control the overview in the widget. So should I still edit it here?

File: src/datamanager.cpp (hunk 2 above)
Comment (sainho93) [reply src/datamanager.cpp]: I haven't decided what to switch here, I have to discuss it with mohcine then complete here

**Pull request merged**